docs/reference/3-admin-api/installation/admin-api-2x-for-odsapi-7x/admin-api-2x-docker-installation.md

```diff
@@ -0,0 +1,263 @@
+# Admin API 2.x - Docker installation
+
+**Contents:**
+
+*   [Before You Install](#before-you-install)
+    *   [Compatibility & Supported ODS / API Versions](#compatibility-supported-ods-api-versions)
+*   [Installation Instructions](#installation-instructions)
+    *   [General Prerequisites](#general-prerequisites)
+*   [Installation Instructions](#installation-instructions)
+    *   [1\. Include Admin API in the ODS Docker Setup](#1-include-admin-api-in-the-ods-docker-setup)
+    *   [2\. Relaunch the Docker Composition](#2-relaunch-the-docker-composition)
+    *   [3\. Execute First-Time Configuration](#3-execute-first-time-configuration)
+
+# Before You Install
+
+This section provides general information you should review before installing the Ed-Fi ODS / API Admin API for v2.2.0.
+
+## Compatibility & Supported ODS / API Versions
+
+This version of the Admin API has been tested and can be installed for use with Ed-Fi ODS / API v7.1. See the [Ed-Fi Technology Version Index](https://edfi.atlassian.net/wiki/spaces/ETKB/pages/20875717/Ed-Fi+Technology+Version+Index) for more details.
+
+# Installation Instructions
+
+## General Prerequisites
+
+The following are required to install the Admin API:
+
+*   The Admin API provides an interface to administer an Ed-Fi ODS / API. Understandably, you must have an instance of the Ed-Fi ODS / API v7.1 deployed and operational before you can use the Admin API. Tested configurations include on-premises installation via [binary installation](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V520/pages/25100419/Getting+Started+-+Binary+Installation) or [source code installation](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V520/pages/25100348/Getting+Started+-+Source+Code+Installation). 
```

Comment:
This page is for Admin API 2, but the link is to ODS/API 5.2. Opportunity to correct that now:

```suggestion
*   The Admin API provides an interface to administer an Ed-Fi ODS / API. Understandably, you must have an instance of the Ed-Fi ODS / API v7.1 deployed and operational before you can use the Admin API. Tested configurations include on-premises installation via [binary installation](/reference/ods-api/getting-started/binary-installation/) or [source code installation](/reference/ods-api/getting-started/source-code-installation/). 
```